biahub/cli/utils.py

```diff
@@ -643,9 +662,39 @@ def _is_nested(lst):
 
 def _check_nan_n_zeros(input_array):
```

Comment:
In this PR, we are entirely changing how this function works - it goes from returning a boolean to returning a mixed output. Is this required for tracking? We should make sure we maintain backward compatibility.

Instead of having three different outputs - boolean (2D), list[int] (3D), list[(int, list)], I would suggest always outputting list[(channel_idx, z_idx)] of the C/Z index that contain a blank 2D image. This will ensure compatibility with the old use case of `if _check_nan_n_zeros(arr)` - the list will be empty if there are no blank images. In case there are blank images, you'd return, for example `[(0, 0)]` (2D), `[(0, 0), (0, 1)]` (3D), or `[(0, 0), (1, 0)]` (4D).